README.md

```diff
@@ -262,12 +262,20 @@ You will see the automatic interactive API documentation (provided by <a href="h
 
 ### Alternative API docs
 
-And now, go to <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>.
+There are two alternatives to Swagger UI: Redocly & Scalar.
```

Comment:
The tool is called "Redoc". Redocly is the company that builds the Redoc tool, right?

Reply:
Also, there are more than two alternatives to Swagger UI, not just Redoc and Scalar, right? Only mentioning it since you mention "There are two alternatives..."

Reply:
Ah apologies, great catch! I just pushed up this fix ✨